app/elements/pages/page-element.html

```diff
@@ -50,6 +50,7 @@ <h2 id="element-title">[[element]]</h2>
               <template is="dom-repeat" items="[[docDemos]]">
                 <a is="app-link" class="item" href="[[_demoLink(active,item.path)]]" active$="[[_demoActive(item.path,view)]]"><iron-icon icon="visibility"></iron-icon> <span>[[_demoName(item.desc)]]</span></a>
               </template>
+              <a class="item" href="[[_githubLink(element)]]" target="_blank"><iron-icon icon="code"></iron-icon> <span>Source</span></a>
```

Comment:
since the element could from a different org other than `PolymerElements` we need to use the source, `href="[[_githubLink(metadata.source)]]"` and _githubLink should return:
```
    _githubLink: function(source) {
      return "https://github.com/" + source;
    },
```